pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -67,6 +69,9 @@ var (
 	LocalCertDir = env.RegisterStringVar("ROOT_CA_DIR", "./etc/cacerts",
 		"Location of a local or mounted CA root")
 
+	useRemoteCerts = env.RegisterBoolVar("USE_REMOTE_CERTS", false,
```

Comment:
nit - not sure we want "remote" as part of this. It can be used in single cluster as well if we don't want to mount. Although I don't know what the benefit of that would be..